test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/TaxEnrolmentSupport.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers
+
+import cats.data.EitherT
+import org.scalamock.scalatest.MockFactory
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.connectors.TaxEnrolmentConnector
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.{EnrolmentRequest, Error}
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
+
+import scala.concurrent.Future
+
+trait TaxEnrolmentSupport {
```

Comment:
I'm not sure any other spec other for anything other than `TaxEnrolmentService` will ever use this stuff since that is the only thing in charge of enroling

Reply:
OK, will rip this out.